docs/getting-started/terminology.md

```diff
@@ -0,0 +1,42 @@
+    - [**Workers**](/docs/infrastructure/workers/index.md): machines that can execute tasks that don't need to be run on the Octopus Server or individual deployment targets.
+    - [**Worker Pools**](/docs/infrastructure/workers/worker-pools.md): a group of workers.  One pool might be in a particular network security zone. Another pool might have a specific set of tools installed.
+    - [**Health Check**](/docs/infrastructure/deployment-targets/machine-policies.md#health-check): a task Octopus periodically runs on deployment targets and workers to ensure that they are available.
+    - [**Machine Policies**](/docs/infrastructure/deployment-targets/machine-policies.md): groups of settings that can be applied to Tentacle and SSH endpoints used for health checks, updating calamari, and more.
+    - [**Machine Proxies**](/docs/infrastructure/deployment-targets/proxy-support.md): allow you to specify a proxy server for Octopus to use when communicating with a Tentacle or SSH Target; you can also specify a proxy server when a Tentacle and the Octopus Server make web requests to other servers.
+    - [**Accounts**](/docs/infrastructure/deployment-targets/index.md#accounts): credential details used during your deployments, including things like username/password, tokens, Azure and AWS credentials, and SSH key pairs.  
+- [**Projects**](/docs/projects/index.md): where the deployment process, configuration variables, runbooks are defined and configured to deploy and manage your software.
+    - [**Deployment Process**](/docs/deployment-process/index.md): the recipe for deploying your software. You define the recipe by adding steps and variables to a project. 
+    - [**Deployment Steps**](/docs/deployment-process/steps/index.md): the specific action (or set of actions) executed as part of the deployment process each time your software is deployed.
+    - [**Release**](/docs/releases/index.md): a snapshot of the deployment process and the associated assets (packages, scripts, variables) as they existed when the release was created. 
+    - [**Variables**](/docs/projects/variables/index.md): a value stored in the Octopus Server for use in different phases of your deployments.  Variables can be scoped to environments, steps, and more.  Variables allow you to have a consistent deployment process across your infrastructure without having to hard-code or manually update configuration settings that differ across environments, deployment targets, channels, or tenants.
+    - [**Manual Interventions**](/docs/deployment-process/steps/manual-intervention-and-approvals.md): the approval step in Octopus Deploy.  Manual interventions can be scoped to specific teams and environments.
+    - [**Runbooks**](/docs/runbooks/index.md): used to automate routine maintenance and emergency operations tasks like infrastructure provisioning, database management, and website failover and restoration.    
+- **Library** - stores items used across multiple projects
```

Comment:
I've suggested artifacts and assets, does artifacts work in this context?
```suggestion
## Library 

The Octopus library stores artifacts and assets that can be used across multiple projects
```

Reply:
I'll update.  We use the term artifact as well (which I will add) in deployments.

The updated line will be:
> The Octopus library is where you store build artifacts and other assets that can be used across multiple projects.